blocks/hero/hero.css

```diff
@@ -67,6 +87,14 @@ main .section.hero-container {
 /* breakpoint-l */
 @media screen and (min-width: 992px) {
   .block.hero {
+    & picture.hero-background.desktop-only {
+      display: unset;
```

Comment:
This should be `display: block` - otherwise it's gets confusing as to how it derives its state.